server/superdesk/macros.py

```diff
@@ -21,6 +22,13 @@ def load_macros():
     """
     try:
         module = app.config['MACROS_MODULE']
+        already_imported = sorted(sys.modules.keys())
+        for t in already_imported:
```

Comment:
just wondering - is there a need for a loop? it should work with `if module in already_imported: ... ` no?